src/lib/cron-tab/custom.tsx

```diff
@@ -4,18 +4,20 @@ interface CustomCronProp {
     onChange(e: string[]): void
     value: string[]
     translate(e: string): string
+    isDisabled?: boolean
 }
 
 const CustomCron: React.FunctionComponent<CustomCronProp> = (props) => {
     const onChange = (e: {target: { value: string}}) => {
+        if(props.isDisabled===true) return;
```

Comment:
Do we really need === true check?
if(props.idDisabled) is enough rt ?

Reply:
Yes, it's enough, no extra type-check is reqd.